### PR TITLE
[update] Post content Type option for post grid,carousel,masonry

### PR DESCRIPTION
**Pull request by amithire1201**
### Description
https://trello.com/c/t1E2belW/49-post-grid-excerpt
Display Post content type setting added in :-

1. Post Grid

1. Post Carousel

1. Post Masonry

### Types of changes
 New feature (non-breaking change which adds functionality) 

### How has this been tested?
Select Post grid and click on Post content in right pane to select display option for post content

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/post/post-grid/edit.js [hunk 1]
```diff
@@ -697,22 +700,43 @@ class UAGBPostGrid extends Component {
 						checked={ displayPostTaxonomy }
 						onChange={ ( value ) => setAttributes( { displayPostTaxonomy: ! displayPostTaxonomy } ) }
 					/>
+					
+				</PanelBody>
+				<PanelBody title={ __( "Post Content Settings" ) } initialOpen={ false }>
 					<ToggleControl
-						label={ __( "Show Excerpt" ) }
+						label={ __( "Post Content" ) }
```
Comment (vrundakansara) [inline src/blocks/post/post-grid/edit.js:707]: Any reason why we changed this label?

File: src/blocks/post/post-grid/edit.js [hunk 2]
```diff
@@ -697,22 +700,43 @@ class UAGBPostGrid extends Component {
 						checked={ displayPostTaxonomy }
 						onChange={ ( value ) => setAttributes( { displayPostTaxonomy: ! displayPostTaxonomy } ) }
 					/>
+					
+				</PanelBody>
+				<PanelBody title={ __( "Post Content Settings" ) } initialOpen={ false }>
```
Comment (vrundakansara) [inline src/blocks/post/post-grid/edit.js:705]: Any reason why we added a new panel for this?

File: dist/blocks/post/class-uagb-post.php [hunk 3]
```diff
@@ -1156,9 +1160,16 @@ public function render_excerpt( $attributes ) {
 			$excerpt = apply_filters( "uagb_single_post_excerpt_{$attributes['post_type']}", $excerpt, get_the_ID(), $attributes );
```
Comment (vrundakansara) [inline dist/blocks/post/class-uagb-post.php:1160]: This new code should be processed before the filter.

File: src/blocks/post/post-grid/edit.js (hunk 1 above)
Comment (amithire1201) [reply src/blocks/post/post-grid/edit.js:707]: Made change just to make more relevant
Shall I change back to old text ?

File: src/blocks/post/post-grid/edit.js (hunk 2 above)
Comment (amithire1201) [reply src/blocks/post/post-grid/edit.js]: Done

File: dist/blocks/post/class-uagb-post.php (hunk 3 above)
Comment (amithire1201) [reply dist/blocks/post/class-uagb-post.php:1166]: Done

File: readme.txt [hunk 4]
```diff
@@ -175,6 +175,7 @@ When you use the Ultimate Addons for Gutenberg along with the free Astra theme,
 * Improvement: Added missing accessibility to all blocks.
 * Fix: Post Carousel - Incorrect Border Radius issue with respect to the theme.
 * Fix: Info Box - Fixed a minor issue with z-index and theme heading.
+* Improvement: Option to Display Post Content Type as Excerpt/Full-Post.
```
Comment (vrundakansara) [inline readme.txt:178]: @amithire1201 - Is this changelog added for the correct release version?

**Pull request merged**